content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -129,4 +130,12 @@ using the attribute `snapshotClassName`.
 Only VolumeSnapshotContents of the requested class, ones with the same `snapshotClassName`
 as the VolumeSnapshot, can be bound to the VolumeSnapshot.
 
+## Provisioning Volumes from Snapshots
+
+A new volume, pre-populated with data from a snapshot, can be provisioned using
+*dataSource* field in the `PersistentVolumeClaim` object.
```

Comment:
Our style guide recommends active voice.
Suggestion:
You can provision a new volume, pre-populated with data from a snapshot, by using the *dataSource* field ...